GeneralsMD/Code/GameEngine/Source/GameClient/GUI/GameWindowTransitions.cpp

```diff
@@ -180,6 +180,10 @@ Bool TransitionWindow::init( void )
 	m_transition = getTransitionForStyle( m_style );
 	m_transition->init(m_win);
 
+	// TheSuperHackers @fix Mauller 15/05/2025 Add TransitionWindow pointer to a vector in the GameWindow so pointers to the GameWindow can be cleared when the GameWindow is destroyed
```

Comment:
I think it is better to not explain GameWindow internals here (see vector comment). Just say we link this TransitionWindow to the GameWindow, so that it can unlink itself from this TransitionWindow when the GameWindow is destroyed.